applications/kommander/0.16.0/defaults/cm.yaml

```diff
@@ -127,3 +127,5 @@ data:
         - "gatekeeper"
         - "kommander-flux"
         - "cilium-hubble-relay-traefik"
+    catalogOciRepositories:
+    - "ghcr.io/nroshanrai/nkp-partner-catalog:2.16"
```

Comment:
This should be `nroshanrai/nkp-partner-catalog`. The registry name and the version should be dynamically calculated by the controller.